search_events_app/models/country.py

```diff
@@ -1,3 +1,15 @@
-class Country:
-    def __init__(self, name):
-        self.name = name
+from django.db import models
+
+
+# Create your models here.
+class Country(models.Model):
+    name = models.CharField(max_length=100)
+    alpha_2_code = models.CharField(max_length=2)
+    alpha_3_code = models.CharField(max_length=3)
+    flag = models.CharField(max_length=200)
```

Comment:
Should a flag be related to an URL? Maybe this link is useful in this case: https://docs.djangoproject.com/en/3.0/ref/models/fields/#urlfield

Reply:
Yes, it is. I didn't know about this kind of django field. That'll be useful!